examples/proc_nutation.py

```diff
@@ -50,7 +49,7 @@
     )
     s.fit()
     fit = s.eval(500)
-    fl.plot(fit, human_units=False)
+    fl.plot(fit, human_units=False, ax=ax3)
```

Comment:
this is why we pass back the Axes object from the function

Reply:
I don't want the integral to be part of the subplots because I'd rather show the FID sliced and phased be the third subplot so that the user can see the final phasing before integrating